### ISSUE TITLE
Extract a Chronix JS-Client.

### ISSUE THREAD

**Issue by FlorianLautenschlager** [ChronixDB/chronix.grafana#22]
We should re-factor the plug-in in a way that we extract the chronix-related code and create a separate chronix js-client (a new repo or chronix.server).

**Comment by defaude** [comment]
Agreed. This client, however, should not be depending on AngularJS ;) Should look & feel more like a library that all "connect-chronix-to-X" plugins can use.

**Comment by FlorianLautenschlager** [comment]
Correct. I suggest, that in a first version we should support two things:
 - querying data
 - facet search
